WordPress/Classes/ViewRelated/Reader/ReaderCardsStreamViewController.swift

```diff
@@ -24,6 +24,9 @@ class ReaderCardsStreamViewController: ReaderStreamViewController {
 
     private var selectInterestsViewController: ReaderSelectInterestsViewController = ReaderSelectInterestsViewController()
 
+    /// Whether the current view controller is visible
+    private var isVisible = false
```

Comment:
I recommend turning this into a computed property
```
private var isVisible: Bool {
 return isViewLoaded && view.window != nil
}
```

Reply:
I've used `isViewOnScreen()` but it didn't work, this one actually works. Done!